docs/source/pre_application/running_a_node.rst

```diff
@@ -64,24 +88,47 @@ the Staking Provider via the ``nucypher bond`` CLI command.
 
 Replace the following values with your own:
 
+   * ``<ETH KEYSTORE URI>`` - The local ethereum keystore (e.g. ``keystore:///root/.ethereum/keystore`` for mainnet)
    * ``<NETWORK NAME>`` - The name of the PRE network (mainnet, ibex, or lynx)
    * ``<L1 PROVIDER URI>`` - The URI of a local or hosted ethereum node (infura/geth, e.g. ``https://infura.io/…``)
    * ``<STAKING PROVIDER ADDRESS>`` - the ethereum address of the staking provider
-   * ``<STAKING PROVIDER KEYSTORE URI>`` - the local ethereum keystore of the staking provider address
    * ``<OPERATOR ADDRESS>`` - the address of the operator to bond
 
 
-.. note::
+via Local Installation
+----------------------
 
-    See ``nucypher bond --help`` for usage.
+.. code:: bash
+
+    (nucypher)$ nucypher bond --signer <ETH KEYSTORE URI> --network <NETWORK NAME> --eth-provider <L1 PROVIDER URI> --staking-provider <STAKING PROVIDER ADDRESS> --operator-address <OPERATOR ADDRESS>
```

Comment:
This is a nice addition.  Not exactly sure where to point users to a web UI at this stage but ultimately that can be the primary place to perform operator bonding.

Reply:
Will be good to mention / highlight which account needs to perform this transaction and why it's required here.  IIRC this can be done by either the staking provider's account, or the stake owner's account?

Reply:
That was done at the beginning of the ``Bond Operator`` section - see L50. Is that sufficient?